app/views/katello/api/v2/subscriptions/show.json.rabl

```diff
@@ -59,10 +59,3 @@ node :activation_keys do |subscription|
     }
   end
 end
-
-node :host, :if => lambda { |sub| sub && sub.host } do |subscription|
```

Comment:
I should clarify I mean that the ```node :host``` from the base rabl template is used, not this one, so its fine to remove it here. Since the type is shown in the UI in an index call, the host is necessary in the base template the way things are set up now. I agree that it would be nice to have a way to avoid this, it would be much faster w/o :)